### PR TITLE
Again on auto normalisation issue

### PR DESCRIPTION
**Pull request by PyrApple**
After several use cases where I need loading without normalisation (latest concerned ILD biquad coefficients), I propose to add a mysofa_open_no_norm method to the easy API (in stead of the previously suggested normalisation flag in #34 for retro compatibility).

*(a commit concerning the dimension of the Data.Delay field introduced itself in the PR, harmless afaik)*

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -40,14 +40,8 @@ if(BUILD_TESTS)
 
   enable_testing()
 
-  add_test(CIPIC_subject_003_hrir_final ../tests/compare.sh ../tests/CIPIC_subject_003_hrir_final)
```
Comment (hoene) [inline CMakeLists.txt]: Please do not remove the test cases.
Instead, please add a file with M,R delays.

File: src/hrtf/easy.c [hunk 2]
```diff
@@ -15,7 +16,7 @@
  *
  */
 
-MYSOFA_EXPORT struct MYSOFA_EASY* mysofa_open(const char *filename, float samplerate, int *filterlength, int *err)
+MYSOFA_EXPORT struct MYSOFA_EASY* mysofa_open_default(const char *filename, float samplerate, int *filterlength, int *err, bool applyNorm)
```
Comment (hoene) [inline src/hrtf/easy.c]: Please define struct MYSOFA_EASY* mysofa_open_default as a static function, not as exported.

**Comment by PyrApple** [comment]
Added cunit tests, do let me know if you think them not thorough enough.

**Comment by hoene** [comment]
Looks good! Thank you.

On the weekend, I will test it and merge it.

**Pull request merged**